glass-easel/src/backend/composed_backend_protocol.ts

```diff
@@ -30,21 +23,17 @@ export interface Context extends Partial<suggestedBackend.Context> {
   createFragment(): Element
   onEvent(
     listener: (
-      target: unknown,
+      target: GlassEaselElement,
       type: string,
       detail: unknown,
       options: EventOptions,
-    ) => EventBubbleStatus,
+    ) => EventBubbleStatus | void,
```

Comment:
Why?

Reply:
> Why?

Developers may simply not returning anything